### PR TITLE
Add single task route

### PR DESCRIPTION
**Pull request by danny-andrews**

### CODE REVIEW

**Comment by Triston117** [comment]
"Hmm, I don't think I will"

File: server/index.js [hunk 1]
```diff
@@ -16,6 +16,14 @@ app.get("/api/tasks", (req, res) => {
   });
 });
 
+app.get("/api/tasks/:id", (req, res) => {
+  const id = req.params.id;
+
+  sql`SELECT * FROM tasks WHERE id = ${id}`.then((rows) => {
```
Comment (timgall) [inline server/index.js:22]: Why in the world would you do this?

File: server/index.js [hunk 2]
```diff
@@ -16,6 +16,14 @@ app.get("/api/tasks", (req, res) => {
   });
 });
 
+app.get("/api/tasks/:id", (req, res) => {
+  const id = req.params.id;
+
```
Comment (gfitzsicat) [inline server/index.js:21]: Will's Youtube

File: server/index.js (hunk 1 above)
Comment (danny-andrews) [reply server/index.js:22]: postgres santizes input.

File: server/index.js [hunk 3]
```diff
@@ -16,6 +16,14 @@ app.get("/api/tasks", (req, res) => {
...
+  sql`SELECT * FROM tasks WHERE id = ${id}`.then((rows) => {
+    res.send(rows);
```
Comment (gfitzsicat) [inline server/index.js:23]: send row[0]

File: server/index.js [hunk 4]
```diff
@@ -16,6 +16,14 @@ app.get("/api/tasks", (req, res) => {
   });
 });
 
+app.get("/api/tasks/:id", (req, res) => {
+  const id = req.params.id;
```
Comment (gfitzsicat) [inline server/index.js:20]: destructure. const { id } = req.params.id;

**Pull request merged**